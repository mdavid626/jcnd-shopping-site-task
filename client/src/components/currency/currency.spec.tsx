import { cleanup, render } from '@testing-library/react';
import React from 'react';
import Currency from './currency';

describe('currency', () => {
  afterEach(cleanup);

  it('should render', () => {
    const { asFragment } = render(<Currency priceInCents={156945} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
