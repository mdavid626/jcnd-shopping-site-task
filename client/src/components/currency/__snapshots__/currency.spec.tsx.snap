// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`currency should render 1`] = `
<DocumentFragment>
  1.569,45 €
</DocumentFragment>
`;
